sympy/core/tests/test_args.py

```diff
@@ -1993,6 +1993,11 @@ def test_sympy__functions__combinatorial__numbers__motzkin():
     assert _test_args(motzkin(5))
 
 
+def test_sympy__functions__combinatorial__numbers__schroeder():
+    from sympy.functions.combinatorial.numbers import schroeder
+    assert _test_args(schroeder(5))
```

Comment:
Does `schroeder(5)` return an integer? If so then this should be changed to something like `schroeder(x)`.